10up/ElasticPress
# [Facet Meta] Compatibility with block themes

### Description of the Change

This PR adds compatibility with block themes for the two Facet meta blocks. It gets all blocks across all block templates and template parts and fields selected in all `elasticpress/facet-meta*` blocks instantiated.

To-Dos:

- [x] Cache layer


Closes #3378 

### How to test the Change

Use a FSE-compatible theme like Twenty Twenty Three, instantiate a Facet Meta block and see it working.

### Changelog Entry

> Added - Compatibility with block themes for the two Facet meta blocks.

### Credits

Props @felipeelia 

### Checklist:
